src/System.Numerics.Vectors/src/System/Numerics/Vector.cs

```diff
@@ -4290,7 +4290,7 @@ internal static unsafe Vector<T> SquareRoot(Vector<T> value)
                     SByte* dataPtr = stackalloc SByte[Count];
                     for (int g = 0; g < Count; g++)
                     {
-                        dataPtr[g] = (SByte)Math.Sqrt((SByte)(object)value[g]);
+                        dataPtr[g] = unchecked((SByte)Math.Sqrt((SByte)(object)value[g]));
```

Comment:
This file and the next one (Vector.cs and GenericVectorTests.cs) are both generated based on .tt files in the same directory. We need to change the .tt file in order for the changes to persist here.

On the other hand, though, I'm not sure there's any real value in turning it on for System.Numerics.Vectors. 90% of the library is floating-point math, and the other 10% feels like it should be unchecked by default due to the nature of `Vector<T>` (the stuff in this file).